### PR TITLE
setting actors aligned

### PR DESCRIPTION
**Pull request by anmol2302**

### CODE REVIEW

**Comment by anmol2302** [comment]
@bvinayakumar  added response class instead of object class

File: actors/systemsettings/src/main/java/org/sunbird/systemsettings/actors/SystemSettingsActor.java [hunk 1]
```diff
@@ -44,9 +43,22 @@ public void preStart() throws Exception {
       if (CollectionUtils.isNotEmpty(settings)) {
         settings
             .stream()
-            .map(f -> cache.put(ActorOperations.GET_SYSTEM_SETTING.getValue(), f.getField(), f))
+            .map(
+                settingObject -> {
+                  Map<String, Object> fieldMap = new HashMap<>();
+                  fieldMap.put(
+                      "field",
```
Comment (bvinayakumar) [inline actors/systemsettings/src/main/java/org/sunbird/systemsettings/actors/SystemSettingsActor.java]: Use existing constant for 'field'.

File: actors/systemsettings/src/main/java/org/sunbird/systemsettings/actors/SystemSettingsActor.java [hunk 2]
```diff
@@ -44,9 +43,22 @@ public void preStart() throws Exception {
...
+                      settingObject.getField()); // preparing request and converting into hash.
+                  String fieldMapToStr = HashGeneratorUtil.getHashCodeAsString(fieldMap);
```
Comment (bvinayakumar) [inline actors/systemsettings/src/main/java/org/sunbird/systemsettings/actors/SystemSettingsActor.java]: fieldMapToStr rename to hashCodeStr

Comment (anmol2302) [reply actors/systemsettings/src/main/java/org/sunbird/systemsettings/actors/SystemSettingsActor.java]: done

File: actors/systemsettings/src/main/java/org/sunbird/systemsettings/actors/SystemSettingsActor.java (hunk 1 above)
Comment (anmol2302) [reply actors/systemsettings/src/main/java/org/sunbird/systemsettings/actors/SystemSettingsActor.java]: done

**Pull request closed without merging**